admin-dev/themes/new-theme/scss/components/_modulescards.scss

```diff
@@ -127,55 +121,39 @@
 * Facebook-like module cards
 **/
 
-@-webkit-keyframes placeHolderShimmer {
-  0% {
-    background-position: -468px 0
-  }
-  100% {
-    background-position: 468px 0
-  }
-}
 @keyframes placeHolderShimmer {
   0% {
-    background-position: -468px 0
+    background-position: -468px 0;
   }
   100% {
-    background-position: 468px 0
+    background-position: 468px 0;
   }
 }
 
 .module-placeholders-wrapper {
+  margin-top: 76px;
+
   .timeline-item-wrapper {
     @extend .module-item-wrapper-grid;
     height: 250px;
 
     .animated-background {
+      position: relative;
+      height: 96px;
+      background: #f6f7f8;
+      background: #eee;
+      background: linear-gradient(to right, #eee 8%, #ddd 18%, #eee 33%);
+      background-size: 800px 104px;
+      animation-name: placeHolderShimmer;
 
-      -webkit-animation-duration: 1.75s;
       animation-duration: 1.75s;
-      -webkit-animation-fill-mode: forwards;
-      animation-fill-mode: forwards;
-      -webkit-animation-iteration-count: infinite;
-      animation-iteration-count: infinite;
-      -webkit-animation-name: placeHolderShimmer;
-      animation-name: placeHolderShimmer;
-      -webkit-animation-timing-function: linear;
       animation-timing-function: linear;
-      background: #f6f7f8;
-      background: #eeeeee;
-      background: -webkit-gradient(linear, left top, right top, color-stop(8%, #eeeeee), color-stop(18%, #dddddd), color-stop(33%, #eeeeee));
-      background: -webkit-linear-gradient(left, #eeeeee 8%, #dddddd 18%, #eeeeee 33%);
-      background: linear-gradient(to right, #eeeeee 8%, #dddddd 18%, #eeeeee 33%);
-      -webkit-background-size: 800px 104px;
-      background-size: 800px 104px;
-      height: 96px;
-      position: relative;
+      animation-iteration-count: infinite;
```

Comment:
No need for `-webkit*` any more? There is no mixin for this that guaranties the compatibility with all browsers?
Or maybe it is automatically added during compilation for this property?

Reply:
I still checked in css files it looks generated, overwise if I'm wrong we should auto generate it, I'm gonna check again